src/test/scala/nl.knaw.dans.easy.bag2deposit/ddm/RewriteSpec.scala

```diff
@@ -296,63 +284,41 @@ class RewriteSpec extends AnyFlatSpec with SchemaSupport with Matchers {
       Success(normalized(expectedDdm))
   }
 
-  it should "complain about invalid period/subject" in {
-    val ddmIn = ddm(
-        <ddm:profile>
-          <dc:title>blablabla</dc:title>
-          { mandatoryInProfile }
-        </ddm:profile>
+  "ddmTransformer" should "add inCollection for archaeology" in {
+    val ddmIn = ddm(title = "blabla", audience = "D37000", dcmi =
         <ddm:dcmiMetadata>
-            <dcterms:temporal xsi:type="abr:ABRperiode">rabarbera</dcterms:temporal>
-            <dc:subject xsi:type="abr:ABRcomplex">barbapappa</dc:subject>
         </ddm:dcmiMetadata>
     )
 
-    cfg.ddmTransformer.transform(ddmIn, "easy-dataset:123").map(normalized) shouldBe
-      Failure(InvalidBagException("temporal rabarbera not found; subject barbapappa not found"))
+    val transformer = cfg.ddmTransformer
+      .copy(collectionsMap = Map("easy-dataset:123" -> <inCollection/>))
+    transformer.transform(ddmIn, "easy-dataset:456").map(normalized) shouldBe Success(normalized(ddmIn))
+    transformer.transform(ddmIn, "easy-dataset:123").map(normalized) shouldBe Success(normalized(
+      ddm(title = "blabla", audience = "D37000", dcmi =
+        <ddm:dcmiMetadata>
+          <inCollection/>
```

Comment:
every ddm will get an empty element? Since i made some attributes mandatory, this is not allowed

Reply:
This unit test just proves _that_ an element is added to a DDM. `CollectionsSpec` proves which elements are configured for which dataset.